python/tox.ini

```
@@ -0,0 +1,67 @@
+[tox]
+skipsdist = True
+envlist =
+    py{39, }
+    lint
+    test
+    coverage
+
+[isort]
+profile = black
+combine_as_imports=True
+force_grid_wrap = 0
+force_sort_within_sections=True
+known_standard_library=pytest
+line_length=21
+multi_line_output=3
+use_parentheses=True
+ensure_newline_before_comments = True
+
+[flake8]
+max-line-length= 101
+exclude= venv*,.tox,docs,build
+ignore=W503, F401
+
+[mypy]
+warn_unused_ignores = True
+check_untyped_defs = True
+disallow_incomplete_defs = True
+disallow_untyped_defs = True
+disallow_any_generics = True
+warn_unused_configs = True
+strict_equality = True
+
+[rstcheck]
+report=warning
+
+[testenv:lint]
+basepython=python3
+skip_install = true
+deps = pre-commit
+commands=
+    pre-commit run --all-files --show-diff-on-failure
+
+[testenv:test]
+whitelist_externals = poetry
```

```suggestion
allowlist_externals = poetry
```
Maybe it's version-related, but I need to change to 'allowlist_external'.